Makefile

```diff
@@ -1,8 +1,19 @@
+## local variables.
+cert_manager_submodule_dir = cert-manager
+cert_manager_operator_submodule_dir = cert-manager-operator
+cert_manager_containerfile_name = Containerfile.cert-manager
+cert_manager_acmesolver_containerfile_name = Containerfile.cert-manager.acmesolver
+cert_manager_operator_containerfile_name = Containerfile.cert-manager-operator
+cert_manager_operator_bundle_containerfile_name = Containerfile.cert-manager-operator.bundle
+commit_sha = $(strip $(shell git rev-parse HEAD))
+source_url = $(strip $(shell git remote get-url origin))
+release_version = v$(strip $(shell git branch --show-current | cut -d'-' -f2))
```

Comment:
Can we use git tags? Especially, for cert-manager operand we should. Operator versions can use either tags or information from ClusterServiceVersion in the `cert-manager-operator/bundle` directory.

Reply:
I think it's not required, anyone wants to build an image locally can make use of the options like `CERT_MANAGER_OPERATOR_IMAGE` , `IMAGE_VERSION` and `IMAGE_BUILD_ARGS`... to initialize with the desired values